src/holochainClient.js

```diff
@@ -34,7 +34,7 @@ async function initAndGetHolochainClient () {
   if (holochainClient) return holochainClient
   try {
     holochainClient = await hcWebClientConnect({
-      url: process.env.REACT_APP_DNA_INTERFACE_URL,
+      url: process.env.NODE_ENV === 'production' ? undefined : process.env.REACT_APP_DNA_INTERFACE_URL,
```

Comment:
this is the same as `url: process.env.REACT_APP_DNA_INTERFACE_URL`

I don't understand why we're doing this. Just let the environment determine it.

Reply:
@robbiecarlton Yes, but this would mean that Everytime you have to make a build you will have to remove your env var from your .env file.
But if u use the NODE_ENV you do not have to deal with this issue. This helps in automation.
Especially when u have to run test as well as make a build, which we are doing on Hydra.